styles/index.css

```diff
@@ -0,0 +1,136 @@
+html, body {
+    width: 90%;
+    height: 100%;
+    background-color: #003A84;
+    margin: 0 auto;
+    text-align: center;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    
+}
+
+header{
+    padding: 30px;
+    color: #edeae4;
+}
+
+.box-container{
+    /* border: 1px solid white; */
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content:space-around;
+    padding: 10px;
+    margin: 10px;    
+}
+
+#temp-title {
```

Comment:
You went a little aggressive with using ids instead of styles, though they do make it so you don't have to deal with selector precedence so I get it.  🙂